md-to-html/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "markdown-it-synapse-lambda",
+  "version": "1.0.1",
+  "description": "Plugin to create a node js lambda to resolve Synapse markdown to html",
+  "repository": "sage-bionetworks/markdown-it-lambda-fct",
+  "type": "module",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "test": "node --experimental-vm-modules ./node_modules/.bin/jest",
+    "start": "node ./bin/www"
+  },
+  "keywords": [
+    "markdown-it-plugin",
+    "markdown-it",
+    "markdown",
+    "Synapse"
+  ],
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "cookie-parser": "^1.4.5",
+    "debug": "^4.3.4",
+    "eslint": "^7.32.0",
+    "eslint-plugin-nodeca": "^1.0.0",
+    "html-to-text": "^8.0.0",
+    "juice": "^7.0.0",
+    "markdown-it": "^12.3.0",
```

Comment:
Consider updating the markdown-it dependency to "^14.1.0", and removing markdown-it-emphasis-alt (unnecessary with latest markdown-it release).  
There may be other dependencies that also should be updated